docs/usage.rst

```diff
@@ -68,35 +66,28 @@ More recent versions of ``ansible-builder`` support multiple tags:
 ``--file``
 **********
 
-To use a definition file named something other than
-``execution-environment.yml``:
+Specifies the execution environment file. To use a file named something other than ``execution-environment.yml``:
 
 .. code::
 
-   $ ansible-builder build --file=my-ee.yml
+   $ ansible-builder build --file=my-ee-def.yml
 
 ``--galaxy-keyring``
 ********************
 
-With more recent versions of Ansible, it is possible to have the ``ansible-galaxy``
-utility verify collection signatures during installation. This requires a keyring to
-be provided (can be built with GnuPG tooling) to use during verification. Provide
-the path to this keyring with the ``--galaxy-keyring`` option. If this option is not
-supplied, no signature verification will be performed. If it is provided, and the
-version of Ansible is not recent enough to support this feature, an error will
-occur during the image build process.
+Specifies a keyring for ``ansible-galaxy`` to use to verify collection signatures during installation. To verify collection signatures:
 
 .. code::
 
    $ ansible-builder create --galaxy-keyring=/path/to/pubring.kbx
    $ ansible-builder build --galaxy-keyring=/path/to/pubring.kbx
 
+If you do not pass this option, no signature verification is performed. If you do pass this option, but the version of Ansible is too old to support this feature, you will see an error during the image build process.
+
 ``--galaxy-ignore-signature-status-code``
 *****************************************
 
-With ``--galaxy-keyring`` set it is possible to ignore certain errors that may occur while verifying collections.
-It is passed unmodified to ``ansible-galaxy`` calls via the option ``--ignore-signature-status-code``.
-See the ``ansible-galaxy`` documentation for more information.
+When ``--galaxy-keyring`` is set, ignores certain errors that may occur while verifying collections. This option is passed unmodified to ``ansible-galaxy`` calls. See the ``ansible-galaxy`` documentation for more information.
```

Comment:
```suggestion
When ``--galaxy-keyring`` is set,  Galaxy ignores certain errors that may occur while verifying collections. This option is passed unmodified to ``ansible-galaxy`` calls. See the ``ansible-galaxy`` documentation for more information.
```

Reply:
But maybe I should move the `when x is set` phrase, to make this clearer.